init-letsencrypt.sh

```diff
@@ -1,80 +1,74 @@
 #!/bin/bash
 
 if ! [ -x "$(command -v docker-compose)" ]; then
-  echo 'Error: docker-compose is not installed.' >&2
-  exit 1
+echo 'Error: docker-compose is not installed.' >&2
+exit 1
 fi
 
-domains=(example.org www.example.org)
+domains=(mydomain.com)
 rsa_key_size=4096
 data_path="./data/certbot"
-email="" # Adding a valid address is strongly recommended
+email="mymail@mail.com" # Adding a valid address is strongly recommended
 staging=0 # Set to 1 if you're testing your setup to avoid hitting request limits
+subscribe_to_eff=1 # Set to 1 if you provided an e-mail address and want to subscribe to EFF mailings
 
 if [ -d "$data_path" ]; then
-  read -p "Existing data found for $domains. Continue and replace existing certificate? (y/N) " decision
-  if [ "$decision" != "Y" ] && [ "$decision" != "y" ]; then
-    exit
-  fi
+read -p "Existing data found for $domains. Continue and replace existing certificate? (y/N) " decision
+if [ "$decision" != "Y" ] && [ "$decision" != "y" ]; then
+	exit
 fi
-
-
-if [ ! -e "$data_path/conf/options-ssl-nginx.conf" ] || [ ! -e "$data_path/conf/ssl-dhparams.pem" ]; then
```

Comment:
@dappsar  Thanks for suggesting these changes, and was submitted a long time ago, but it looks like you added samples of the options-ssl-nginx.conf and ssl-dhparams.pem file, but then removed all reference to them in your version.  (At least I can't find where they are included, I don't see direct reference to them and I don't see a wildcard (/*.conf) include.